examples/tour/src/main.rs

```diff
@@ -55,6 +62,7 @@ pub enum Message {
     ToggleTextInputIcon(bool),
     DebugToggled(bool),
     TogglerChanged(bool),
+    ToggleBehavior(Toggle)
```

Comment:
probably just set this to a bool same as the others and remove toggle enum

Reply:
sure, will do